### PR TITLE
Add arrow-right icon to new-prod-boxes

### PR DESCRIPTION
**Pull request by Matei-Iordache**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--www-landing-pages--bitdefender.hlx.page/drafts/test-page/
- After: https://DEX-17049--www-landing-pages--bitdefender.hlx.page/drafts/test-page/

### CODE REVIEW

File: _src-lp/blocks/new-prod-boxes/new-prod-boxes.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default function decorate(block) {
 
           // Extract the content of the first <td> to be placed outside the <li>
           let firstTdContent = tdList.length > 0 && tdList[0].textContent.trim() !== '' ? `${tdList[0].innerHTML}` : '';
-
+          console.log(`td content ${firstTdContent}`);
```
Comment (vradulescu-bd) [inline _src-lp/blocks/new-prod-boxes/new-prod-boxes.js]: remove

File: _src-lp/blocks/new-prod-boxes/new-prod-boxes.js [hunk 2]
```diff
@@ -42,6 +42,11 @@ export default function decorate(block) {
             firstTdContent = firstTdContent.replace('&lt;-', '');
           }
 
+          if (firstTdContent.indexOf('-&gt;') !== -1 || firstTdContent.indexOf('&gt;') !== -1) {
```
Comment (vradulescu-bd) [inline _src-lp/blocks/new-prod-boxes/new-prod-boxes.js:47]: those 2 conditions are strange, what do they mean exactly ? maybe extract above in a one or two different simple and verbose expressions

File: _src-lp/blocks/new-prod-boxes/new-prod-boxes.css [hunk 3]
```diff
@@ -168,6 +168,15 @@
   background-size: 15px 12px;
   float: left;
 }
+.new-prod-boxes-container .block .pod_box .inner_pod_box .benefitsLists ul li:first-of-type.has_arrow_right .arrow-right {
```
Comment (vradulescu-bd) [inline _src-lp/blocks/new-prod-boxes/new-prod-boxes.css:171]: pod_box ? maybe prod-box ?

File: _src-lp/blocks/new-prod-boxes/new-prod-boxes.js (hunk 1 above)
Comment (Matei-Iordache) [reply _src-lp/blocks/new-prod-boxes/new-prod-boxes.js]: Solved

File: _src-lp/blocks/new-prod-boxes/new-prod-boxes.js (hunk 2 above)
Comment (Matei-Iordache) [reply _src-lp/blocks/new-prod-boxes/new-prod-boxes.js:47]: Added comments for clarification

**Review by enake — APPROVED** [review]

**Review by vradulescu-bd — APPROVED** [review]

**Pull request merged**